### PR TITLE
Added fluent icons for drives

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
- For #4405, #4429

**Details of Changes**
This PR implements some of the drive icons.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

File: Files/Filesystem/LocationItem.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace Files.Filesystem
 {
     public class LocationItem : ObservableObject, INavigationControlItem
     {
-        public string Glyph { get; set; }
+        //public string Glyph { get; set; }
```
Comment (yaichenbaum) [inline Files/Filesystem/LocationItem.cs:12]: Can this be removed?

Comment (d2dyno1) [reply Files/Filesystem/LocationItem.cs]: @yair100 Done!

File: Files/UserControls/Widgets/DrivesWidget.xaml [hunk 2]
```diff
@@ -186,11 +186,11 @@
                                     <ColumnDefinition Width="Auto" />
                                     <ColumnDefinition Width="*" />
                                 </Grid.ColumnDefinitions>
-                                <FontIcon
+                                <Image
                                     Grid.Column="0"
                                     x:Phase="1"
-                                    FontSize="28"
-                                    Glyph="{x:Bind Glyph}" />
+                                    Height="28"
```
Comment (yaichenbaum) [inline Files/UserControls/Widgets/DrivesWidget.xaml:192]:
```suggestion
                                    Height="40"
```

**Review by yaichenbaum — APPROVED** [review]
Really nice work!

**Pull request merged**